testing/evaluate_tests.py

```diff
@@ -120,12 +125,7 @@ def test_schema(resp, test_value, test_name=""):
     ):
         errs.update(res)
 
-    if errs:
-        logger.info(f"Test {test_name} - Schema test failed for {test_value} with response {errs}")
-        logger.error(test_name)
-        for key in errs.keys():
-            logger.error(f"Expected {key}: {test_value[key]}")
-            logger.error(f"Observed {key}: {errs[key]}\n")
+    print_errors(test_name, test_value, errs)
```

Comment:
i think you could also keep the code in here
and do

```
if errs:
  logger.info(f"Test {test_name} - Schema
  logger.error(test_name)

for key in errs.keys():
  logger.error(f"Expected {key}: {test_value[key]}")
  logger.error(f"Observed {key}: {errs[key]}\n")
```

if errs is {}, the for loop would not log anything

Reply:
I made a print function and moved it to another file